django/core/checks/security/csp.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+
+from .. import Tags, Warning, register
+
+W026 = Warning(
+    "Each enabled CSP setting (SECURE_CSP, SECURE_CSP_REPORT_ONLY) must be a "
+    "dictionary.",
```

Comment:
Can we put the setting name in the message? Otherwise, it leaves the user hunting for where the mistake may be. I'm thinking of complex settings configurations where production has a report-only policy separate from some base configuration.